tests/development-private-key.asc

```diff
@@ -0,0 +1,82 @@
+-----BEGIN PGP PRIVATE KEY BLOCK-----
```

Comment:
any reason we want to keep the private key? is it just for testing?

Reply:
Yes, for testing